hibernate-core/src/test/java/org/hibernate/orm/test/query/CteTests.java

```diff
@@ -241,6 +242,7 @@ public void testSubquery(SessionFactoryScope scope) {
 
 	@Test
 	@SkipForDialect(dialectClass = SybaseASEDialect.class, reason = "The emulation of CTEs in subqueries results in correlation in nesting level 2, which is not possible with Sybase ASE")
+	@SkipForDialect(dialectClass = InformixDialect.class)
```

Comment:
Would be nice to capture the reason why this doesn't work.

Reply:
I've identified the issue with Informix. Simply changing the name to something different from the original table will suffice.